kolibri/core/content/test/test_zipcontent.py

```diff
@@ -99,9 +99,16 @@ def _get_file(self, file_name, base_url=None, **kwargs):
         return generate_zip_content_response(self.environ)
 
     def test_zip_file_url_reversal(self):
+        from kolibri.utils.conf import OPTIONS
+
+        path_prefix = OPTIONS["Deployment"]["ZIP_CONTENT_URL_PATH_PREFIX"]
+
+        if path_prefix != "/":
+            path_prefix = "/" + path_prefix
```

Comment:
A pattern I sometimes use that would be even more tolerant/robust here is: `"/" + path_prefix.strip("/") + "/"`

Reply:
The `.strip` already happens during option validation.